sdks/python/apache_beam/examples/wordcount.py

```diff
@@ -108,12 +108,13 @@ def _add_argparse_args(cls, parser):
   # Actually run the pipeline (all operations above are deferred).
   result = p.run()
   result.wait_until_finish()
-  empty_lines_filter = MetricsFilter().with_name('empty_lines')
-  query_result = result.metrics().query(empty_lines_filter)
-  if query_result['counters']:
-    empty_lines_counter = query_result['counters'][0]
-    logging.info('number of empty lines: %d', empty_lines_counter.committed)
-  # TODO(pabloem)(BEAM-1366): Add querying of MEAN metrics.
+  if hasattr(result, 'has_job') and result.has_job:
```

Comment:
`hasattr(result, 'has_job') and result.has_job` would skip the metric checking for `DirectRunner` as well. This should work for `DirectRunner` and other runner when there are no templates.

Maybe this will do what you want?:
`if not hasattr(result, 'has_job') and result.has_job:`

Also please add a comment about why this is needed.